intro-project_management.Rmd

```diff
@@ -0,0 +1,259 @@
+---
+title: "Best practices in data analysis workflow and project management in R"
+output:
+    html_document:
+        toc: false
+params:
+    datetime: "2018-11-09"
+    level: "Beginner" # or intermediate or advanced
+    videolink: "" # Keep empty if no link yet.
+    instructor: "Luke Johnston"
+---
+
+## Session details
+
+- **Date of session**: `r params$datetime`
+- **Instructor**: `r params$instructor`
+    - **Contributions from**: Maria Izabel Cavassim Alves ([\@izabelcavassim](https://github.com/izabelcavassim))
+- **Session level**: `r params$level`
+    - **Part of the ["Beginner Series"](index.html)**
+- **Required packages to install**:
+
+
+### Objectives
+
+1. To use and learn the best practices for project organization.
+1. To use RStudio to create and manage projects with consistent layout.
+
+**At the end of this session you will be able**:
+
+- To apply the best practices using R.
+- To create a new Rstudio project.
+- To organise folders in a consistent, structured and systematic way.
+
+
+## Resources for learning and help
+
+**For learning**:
+
+- [Good enough practicies in scientific computing ](https://github.com/swcarpentry/good-enough-practices-in-scientific-computing/blob/gh-pages/good-enough-practices-for-scientific-computing.pdf)
```

Comment:
Better to use the DOI  https://doi.org/10.1371/journal.pcbi.1005510

Reply:
Probably good to add the "Best practices" version by the same group  https://doi.org/10.1371/journal.pbio.1001745